docs/source/tutorials/interactive_simulation.rst

```diff
@@ -0,0 +1,185 @@
+
+In this tutorial, we'll walk through getting a simulation up and running in an interactive setting such as a python
+interpreter or Jupyter notebook.  Running a simulation in this way is useful for a variety of reasons, foremost for
+debugging and setting up validation simulations. It allows for changing a configuration of a simulation
+programmatically, stepping through a simulation at will, and examining the state of the simulation itself. For the
+following tutorial, we will assume you have set up an environment and installed vivarium. If you have not, please see
+<section>
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+**Other Relevant Tutorials:**
+
+- To learn more about disease modeling and Vivarium simulations, see <section>
+- To go in depth and explore an Interactive simulation context, see <section>
+
+Setting up a Simulation
+=======================
+
+To run a simulation interactively, we will need to create a simulation object and furnish it with configurations,
+components and (optionally) plugins -- all the things that make up a simulation. The ``Vivarium`` framework provides
+four functions to help us get started with this, all found in ``vivarium_public_health``. They differ along two axes --
+how we give the simulation information about the components, plugins and configurations we'd like to simulate, and
+whether the simulation context is ``setup`` or not. However, each of these methods returns the same
+``InteractiveContext`` simulation object and can be interacted with in the same way.
+
+.. note:
+
+    see <section> for more on the difference between initialization and setup.
+
+The following examples will use the non-setup versions, but be aware that the counterparts exist that will call setup
+for you. Practically speaking, the utility of initializing without setting up is that it allows you to alter the
+configuration of a simulation before it is used. This is frequently useful for setting up specific configurations for
+validation from a notebook, for example. Changing the configuration at the top of the notebook makes it clear what is
+going on and guarantees that a configuration parameter is set as it should be.
+
+With a Model Specification File - The Automatic Way
+---------------------------------------------------
+
+The model specification file contains all the information needed to prepare and run a simulation so we need only
+provide this for one way to get up and running quickly.  We will use the model specification from our disease model
+examples:
+
+.. literalinclude:: ../../../src/vivarium/examples/disease_model/disease_model.yaml
+    :caption: **File**: :file:`disease_model.yaml`
+
+We can prepare and run a simulation interactively with this specification as follows. First, we initialize the
+simulation and get back an ``InteractiveContext`` object.
+
+.. code-block:: python
+
+    from vivarium_public_health.interactive import initialize_simulation_from_model_specification
```

Comment:
this is a tutorial in vivarium so I think we want to use the vivarium versions of these functions not vph

Reply:
lol yes I need to swap those

Reply:
I got caught by our own split, I forget it exists in two places

Reply:
yeah tbh I'm not sure it's worth having a second version in vph. the only difference between the vph and the vivarium functions is that vph adds the artifact plugins by default but you still need to have the plugin block in your model spec for running on the command line/distributed so I don't know that anyone has actually been using that feature